docs/changelog.rst

```diff
@@ -15,6 +15,7 @@ New fields for existing flavors:
 Modifications to existing flavors:
 
 - Fix `FRNationalIdentificationNumber` validation for people born overseas
+- Updated Indian states and union territories names and code as per iso 3166 (https://www.iso.org/obp/ui/#iso:code:3166:IN)
```

Comment:
This change will break some sites since some keys are changing or being removed. We try to follow the standards so this change is ok but it should be documented. All keys that have changed or have been removed should be listed in the changelog entry. I wrote this up quickly:

```
Breaking data changes: The key for Chattisgarh has been changed from CG to CT, the key for Uttarakhand has been changed from UA to UT and the keys DD (Dadra and Nagar Haveli) and  DN (Daman and Diu) hav been remove and combined into DH (Dadra and Nagar Haveli and Daman and Diu).
```

Could you double check that it makes sense and add it to the changelog? Thanks!

Reply:
@benkonrath Thank you for the review. I have added all the changes in the changelog as you suggested. Kindly let me know if I need to add anything further.

Reply:
Looks good, thanks!